src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const price = 40;
+  const discountMoreTwoDays = 20;
+  const discountMoreSevenDays = 50;
+  let result = 0;
+
+  result = days * price;
```

Comment:
You can set days * price value by default to result variable
```suggestion
  const result = days * price;
```